jsonrpc/mocks_test.go

```diff
@@ -220,3 +224,17 @@ func (m *mockStore) GetStateSyncProof(stateSyncID uint64) (types.Proof, error) {
 func (m *mockStore) FilterExtra(extra []byte) ([]byte, error) {
 	return extra, nil
 }
+
+func setupSecretsManagerWithKey(t require.TestingT) *secrets.SecretsManagerMock {
+	ecdsaKey, err := wallet.GenerateKey()
```

Comment:
Make this a test helper function
```suggestion
    t.Helper()
	ecdsaKey, err := wallet.GenerateKey()
```